app/templates/buyers/dashboard.html

```diff
@@ -6,11 +6,21 @@
 {% endblock %}
 
 {% block breadcrumb %}
-  {%
-    with items = breadcrumbs
-  %}
-    {% include "toolkit/breadcrumb.html" %}
-  {% endwith %}
+  {{ govukBreadcrumbs({
+    "items": [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": url_for("buyers.buyer_dashboard"),
+        "label": "Your account"
+      },
+      {
+        "text": "Your requirements"
+      }
+    ]
```

Comment:
Seems to be a problem with this one?

![image](https://user-images.githubusercontent.com/503614/72893649-77b04300-3d11-11ea-804f-942070a268cc.png)

Reply:
fixed with the incoming commit